raster/src/main/scala/geotrellis/raster/render/ColorRamp.scala

```diff
@@ -125,8 +125,21 @@ object ColorRamp {
     * @param n       Length of list to return
     */
   def spread(colors: Vector[Int], n: Int): Vector[Int] = {
+    /* The requested spread matches the original */
     if (colors.length == n) return colors
 
+    /* 2016 May 20 @ 14:21
```

Comment:
A personal habit. I like to timestamp comments, not necessarily the change that was made.